### PR TITLE
Fixed some mistakes

### PR DESCRIPTION
**Pull request by SashaViun**
- [DEMO LINK](https://SashaViun.github.io/layout_stop-watch/)

### CODE REVIEW

**Comment by PolinaSukhorukova** [comment]
Good job! 🔥

**Review by PolinaSukhorukova — CHANGES_REQUESTED** [review]
Second hand unnaturally slows down at the end of the minute.  Try to add animation-timing-function.

File: src/styles/blocks/watch.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @include clock;
+    height: $center-size;
+    width: $center-size;
+  }
+
+  &__seconds {
+    @include design;
+    width: 1%;
+    height: 36%;
+    top: 14%;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/watch.scss:25]: it's not clear what these value means
will be better to calculate them somehow depending on the size of watch / hands

File: src/styles/blocks/watch.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    animation-name: seconds;
+    animation-duration: 60s;
+    animation-iteration-count: 60;
+  }
+
+  &__minutes {
+    @include design;
+    height: 30%;
+    width: 2%;
+    top: 20%;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/watch.scss:36]: same here

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Why did you set border-radius 45% for the main container? let's make watch the circle, not some other figure

File: src/styles/utils/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@keyframes minutes {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes seconds {
```
Comment (alexandra-protyanova) [inline src/styles/utils/_variables.scss:7]: You have 2 absolutely the same keyframes, you can use one of them and nothing will change

**Review by alexandra-protyanova — APPROVED** [review]
Good job 👍

**Pull request closed without merging**